ci/jenkins/run-tests

```diff
@@ -0,0 +1,101 @@
+                dir('liberty-tools-eclipse') {
+                    catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+                        script {
+                            try {
+                                wrap([$class: 'Xvfb', installationName: 'gui-test', autoDisplayName: true, debug: true, parallelBuild: true, screen: '1680x1050x24']) {
+                                    sh "metacity --sm-disable --replace &"
+                                    sh "mvn clean verify -Dtycho.disableP2Mirrors=true -Declipse.target=4Q2022 -Dosgi.debug=./tests/resources/ci/debug.opts -DmvnImportWait=60000 -Dtycho.showEclipseLog -e"
+                                }
+                            } finally {
+                                sh "cd tests/target/surefire-reports && zip -r 4Q2022-test-reports.zip ."
+                                archiveArtifacts artifacts: 'tests/target/surefire-reports/4Q2022-test-reports.zip', fingerprint: true
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        
+        stage('Test on Eclipse 3Q2022') {
+            steps {
+                dir('liberty-tools-eclipse') {
+                    catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+                        script {
+                            try {
+                                wrap([$class: 'Xvfb', installationName: 'gui-test', autoDisplayName: true, debug: true, parallelBuild: true, screen: '1680x1050x24']) {
+                                    sh "metacity --sm-disable --replace &"
+                                    sh "mvn clean verify -Dtycho.disableP2Mirrors=true -Declipse.target=3Q2022 -Dosgi.debug=./tests/resources/ci/debug.opts -DmvnImportWait=60000 -Dtycho.showEclipseLog -e"
+                                }
+                            } finally {
+                                sh "cd tests/target/surefire-reports && zip -r 3Q2022-test-reports.zip ."
+                                archiveArtifacts artifacts: 'tests/target/surefire-reports/3Q2022-test-reports.zip', fingerprint: true
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        
+        stage('Test on Eclipse 1Q2023') {
+            steps {
+                dir('liberty-tools-eclipse') {
+                    catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+                        script {
+                            try {
+                                wrap([$class: 'Xvfb', installationName: 'gui-test', autoDisplayName: true, debug: true, parallelBuild: true, screen: '1680x1050x24']) {
+                                    sh "metacity --sm-disable --replace &"
+                                    sh "mvn clean verify -Dtycho.disableP2Mirrors=true -Declipse.target=1Q2023 -Dosgi.debug=./tests/resources/ci/debug.opts -DmvnImportWait=60000 -Dtycho.showEclipseLog -e"
```

Comment:
Why do we do:  `-Dtycho.disableP2Mirrors=true`  ?  Did we run into trouble?

Reply:
Yea we hit issues without setting this.  I cant remember the exact details, but this didnt seem to impact performance or anything and got past the problem (whatever it was :-) )